test/focus-group.test.ts

```diff
@@ -3,7 +3,8 @@ import { equal } from 'node:assert'
 import { test } from 'node:test'
```

Comment:
Let’s also use new roles in `test/demo`

Reply:
I am a bit stack with the demo. Probably the `listbox` doesn't work properly. I will return tomorrow.

Reply:
I think that we need to discuss `tablist` demo... Do we need to implement full tabs functionality (e.g. https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/) or some "mock" functionality? 

I've added the draft component to index.txs

Reply:
Yes, let’s add tab content (just `name + 'content'`) but add also a focus jump.

So when I will press enter, it will move cursor to the tab block.

(Everything else looks great and ready to merge)

Reply:
Great! I will improve the demo!

Reply:
@ai I improved the `tablist` demo but without the "`enter` key functionality". If I understand the w3.org example correctly it should work without `enter`:

- to focus tabs we use `tab`
- to move between tabs we use `left-arrow` and `right-arrow`
- to move to the content we use `tab` again
- to move to tabs back use `shift + tab` combination

I believe that it would be better to have it as the default behavior. 

The "Enter" mode assumes that the tab doesn't change automatically on focus. Probably it could be another option. What do you think?

Reply:
> The "Enter" mode assumes that the tab doesn't change automatically on focus. Probably it could be another option. What do you think?

Both ways are good